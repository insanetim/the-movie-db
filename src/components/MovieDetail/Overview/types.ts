import { IMovie } from 'src/interfaces/movie.interface'

export type OverviewProps = {
  overview: IMovie['overview']
}
