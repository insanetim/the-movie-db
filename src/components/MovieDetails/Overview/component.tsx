import { Col, Typography } from 'antd'

import { OverviewProps } from './types'

const Overview: React.FC<OverviewProps> = ({ overview }) => {
  return (
    <Col span={24}>
      <Typography.Title level={3}>Overview</Typography.Title>
      <Typography.Paragraph>{overview}</Typography.Paragraph>
    </Col>
  )
}

export default Overview
