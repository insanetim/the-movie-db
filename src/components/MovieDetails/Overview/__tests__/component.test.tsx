import { render } from '@testing-library/react'

import Overview from '../component'

describe('Overview component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Overview overview='test/overview' />)

    expect(asFragment()).toMatchSnapshot()
  })
})
