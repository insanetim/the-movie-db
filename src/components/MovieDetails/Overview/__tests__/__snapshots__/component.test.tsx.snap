// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="ant-col ant-col-24 css-dev-only-do-not-override-1kuana8"
  >
    <h3
      class="ant-typography css-dev-only-do-not-override-1kuana8"
    >
      Overview
    </h3>
    <div
      class="ant-typography css-dev-only-do-not-override-1kuana8"
    >
      test/overview
    </div>
  </div>
</DocumentFragment>
`;
